src/bin/www.ts

```diff
@@ -0,0 +1,20 @@
+import Koa = require('koa');
+import {productsRouter} from '../routers/product-router';
+import bodyParser = require('koa-bodyparser');
+import {loggingMiddleware} from '../middlewares/logging-middleware';
+import {config} from '../../configurations/configuration';
+import {errorHandlerMiddleware} from '../middlewares/error-handler';
+import {connectToDB} from '../actions/db-connector';
+
+
+const app = new Koa();
+connectToDB();
+
+app
```

Comment:
create an `app.ts` in the `src` directory that creates app and adding all middlewares and exports the app instance - its a convention

Reply:
done